src/client/components/Molecules/StartPageButtons/StartPageButtons.jsx

```diff
@@ -0,0 +1,27 @@
+//import CurvedButton from '../Atoms/CurvedButton' (button import, check file path)
+
+export const StartPageButtons = props => {
```

Comment:
Simple declare your variable and export it at the end of your code.

F.ex.
`export { FormInput }`